src/vcsparser.bugdatabase.azuredevops/JSONQuery.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace vcsparser.bugdatabase.azuredevops
+{
+    //public class JSONQueryColumn
+    //{
+    //    public string ReferenceName { get; set; }
+    //    public string Name { get; set; }
+    //    public string Url { get; set; }
+    //}
+
+    public class JSONQueryItem
+    {
+        public string Id { get; set; }
+        public Uri Url { get; set; }
+    }
+
+    public class JSONQuery
+    {
+        //public string QueryType { get; set; }

```

Comment:
Can you please remove these comments?

Reply:
Removed in 436638d125bb40b11223df403d8dd9b234d55800